packages/featureflags/featureToggleManager.js

```diff
@@ -57,6 +57,7 @@ class FeatureToggleManager {
             throw new Error('FeatureToggleManager config error!')
         } catch (err) {
             logger.error({ msg: 'fetchFeatures error', err })
+            return {}
```

Comment:
Maybe try to use cached value?
```suggestion
            const cachedFeatureFlags = await this.redis.get(this._redisKey)
            return !!cachedFeatureFlags ? JSON.parse(cachedFeatureFlags) : {}
```